lib/linguist/languages.yml

```diff
@@ -5945,8 +5945,9 @@ Tcl:
   type: programming
   color: "#e4cc98"
   extensions:
-  - ".tcl"
```

Comment:
From [the top of `languages.yml`](https://github.com/github/linguist/blob/7efa7b3380ad0221c499f9ef1ca84c4dfcc0b299/lib/linguist/languages.yml#L17-L19):

> _the first one is considered the primary extension, the others should be listed alphabetically_

I know it looks jarring, but the `extensions` list should be

~~~yaml
   extensions:
   - ".tcl"
   - ".adp"
   - ".tcl.in"
   - ".tm"
~~~

Reply:
No problem, I will fix that right away. I was confused by the non-regression test suite that told me it was not alphabetically sorted but I miss the point about the primary extension in that list.

Reply:
You're right, that could probably be made clearer. I'll see if I can improve the feedback generated by `test_pedantic.rb` in a later PR, especially since the _`X should come after 𝑌`_ messages quickly become annoying when multiple extensions aren't sorted.